src/components/UserInfo.js

```diff
@@ -1,16 +1,20 @@
 export default class UserInfo {
-  constructor({ nameSelector, jobSelector }) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._jobElement = document.querySelector(jobSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     return {
       name: this._nameElement.textContent,
       job: this._jobElement.textContent,
+      avatar: this._avatarElement,
```

Comment:
Needs correction: Please add a separate getter for the avatar, since it's controlled by a different form